src/hotspot/share/gc/g1/g1FullGCCompactTask.hpp

```diff
@@ -50,7 +50,7 @@ class G1FullGCCompactTask : public G1FullGCTask {
 
   class G1CompactRegionClosure : public StackObj {
     G1CMBitMap* _bitmap;
-
+    void clear_bitmap(oop object);
```

Comment:
Good point, I'll update the name, but will leave the "par"-part out as this is not done with a par-operation. Since the whole region is handled by a single worker the clearing is done with a `_bm.clear_bit()` and I can't come up with a case where this would be a problem. The regions and bitmap should be aligned so that there are no words in the bitmap spanning two regions, right?